src/styles/blocks/review.scss

```diff
@@ -0,0 +1,15 @@
+.review {
+  display: flex;
+  flex-direction: row;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), if you are using `flex-direction: row;`, it is unnecessary because it's the default value. You can remove this line unless you have a specific reason to include it for clarity.